ClientPlugin/Config.cs

```diff
@@ -0,0 +1,57 @@
+using ClientPlugin.Settings.Elements;
+using Sandbox.Graphics.GUI;
+using System;
+using System.Text;
+using System.Xml.Serialization;
+using VRage.Input;
+using VRageMath;
+
+
+namespace ClientPlugin.Settings
+{
+    public enum DropDownEnum
+    {
+        Alpha,
+        Beta,
+        Gamma,
+        Delta,
+        Epsilon
+    }
+
+    public class Config
```

Comment:
For reusing the same config object later for the DS and Torch we need `Config` to have `INotifyPropertyChanged`. It would allow for unifying the config for all three targets. It would also allow for plugin components to listed on config change wherever it is applicable/relevant.

Reply:
Done in https://github.com/sepluginloader/PluginTemplate/pull/22/commits/be0af5b20755e25adb790a11119fffa973577459
It is worth noting this makes the user's work significantly more complicated.

Reply:
You're right. Let's find a way to simplify the user's work in a separate PR. E.g. providing similar functionality without having to write setters, if that's possible.